app/models/cedar-metadata-record.ts

```diff
@@ -0,0 +1,32 @@
+import { AsyncBelongsTo, attr, belongsTo } from '@ember-data/model';
+import CedarMetadataTemplateModel from 'ember-osf-web/models/cedar-metadata-template';
+import AbstractNodeModel from 'ember-osf-web/models/abstract-node';
+import FileModel from 'ember-osf-web/models/file';
+import { tracked } from '@glimmer/tracking';
+
+import OsfModel from './osf-model';
+
+
+/**
+ * CedarMetadataRecordModel
+ *
+ * @description The cedar metadata record model
+ */
+export default class CedarMetadataRecordModel extends OsfModel {
+    @attr('object') metadata!: any;
+    @attr('boolean') isPublished!: boolean;
+
+    @tracked templateName!: string;
```

Comment:
Note: Thought this was unused and leftover, but this is used as an alias for the `template.name` in the code.